.github/workflows/TriggerReferenceDocsCI.yml

```diff
@@ -25,56 +25,24 @@ jobs:
       env: 
         AdoOrg: ${{secrets.ADO_DocsReference_Organization}}
         AdoProject: ${{secrets.ADO_DocsReference_Project}}
-        AdoPipelineId: ${{secrets.ADO_DocsReference_JumpPipeline_ID}}
-        CheckInterval: 60
+        AdoLatestPipelineId: ${{secrets.ADO_DocsReference_Latest_Pipeline_ID}}
+        AdoLtsPipelineId: ${{secrets.ADO_DocsReference_LTS_Pipeline_ID}}
+        ReleaseBranch: ${{ github.event.release.target_commitish }}

```

Comment:
Please confirm the trigger event's `${{ github.event.release.target_commitish }}` and event type for LTS-doc  Pipeline